reference/forms/types/options/extra_fields_message.rst.inc

```diff
@@ -7,3 +7,5 @@ This is the validation error message that's used if the submitted form data
 contains one or more fields that are not part of the form definition. The
```

Comment:
Here, something like:
```rst
.. versionadded: 5.1

    The support for pluralization has been added in Symfony 5.1.
```

See https://symfony.com/doc/current/contributing/documentation/format.html#new-features-behavior-changes-or-deprecations, to learn more about our standards.

Reply:
Ah I cannot comment on line 3.

Reply:
Thanks for your feedback @HeahDude I will re-read the guidelines to make your life easier next time :p

I just added the required information.

Reply:
>I will re-read the guidelines to make your life easier next time

No worries ;)